### PR TITLE
Input Details from user end

### PR DESCRIPTION
**Pull request by JazibEqbal**

### CODE REVIEW

File: src/screens/Details.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { Category, CategoryRank, Gender, Rank, IsPWD, State, Branches } from '../components/DetailsComponents';
+
+function Formm() {
```
Comment (imf-ali) [inline src/screens/Details.js:4]: Check function name

File: src/screens/Details.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    categoryRank: '',
+    gender: '',
+    ispwd: '',
+    state: '',
+    branch: ''
+  });
+
+  const FormTitles = ['Enter your rank', 'Please choose your category', 'Please select your gender', 'Category Rank if applicable', 'Are you physically disabled?', 'Enter your State', 'Please select preferred Branch'];
+
+  const PageDisplay = () => {
```
Comment (imf-ali) [inline src/screens/Details.js:18]: Use switch case for multiple else if

Comment (JazibEqbal) [reply src/screens/Details.js:18]: kaam ho raha h isse bhi

Comment (imf-ali) [reply src/screens/Details.js:18]: Fix it. This is not how a code is done

File: src/screens/Details.js [hunk 3]
```diff
@@ -15,21 +15,15 @@ function Details() {
...
-      return <CategoryRank formData={formData} setFormData={setFormData} />;
-    } else if (page === 4) {
-      return <IsPWD formData={formData} setFormData={setFormData} />;
-    } else if (page === 5) {
-      return <State formData={formData} setFormData={setFormData} />;
-    } else {
-      return <Branches formData={formData} setFormData={setFormData} />;
+  const SwitchPage = () => {
+    switch (page) {
+      case 0: return <Rank formData={formData} setFormData={setFormData} />;
```
Comment (imf-ali) [inline src/screens/Details.js:20]: Its missing break, at the end of every case

**Review by imf-ali — APPROVED** [review]
LGTM!

**Pull request merged**